pkg/assetfetcher/http.go

```diff
@@ -100,12 +100,12 @@ func (c *NexposeAssetFetcher) FetchAssets(ctx context.Context, siteID string) (<
 	pagedErrChan := make(chan error, siteAssetResp.Page.TotalResources)
 	for _, asset := range siteAssetResp.Resources {
 		if !asset.hasBeenScanned() {
-			// no need to continue processing a Nexpose Asset that has never been scanned
-			stater.Count("assetreceived.skipped", 1)
+			stater.Count("assetmissingfield", 1, fmt.Sprintf("site:%s", siteID), "TimeStamp")
 			continue
 		}
 		assetEvent, err := asset.AssetPayloadToAssetEvent()
 		if err != nil {
+			stater.Count("assetmissingfield", 1, fmt.Sprintf("site:%s", siteID), asset.AssetMissingField())
```

Comment:
suggestion: We're already sending an error down the error channel to account for this case. It is the recipient of those errors that should be emitting metrics and not this component.